roktux/src/main/java/com/rokt/roktux/component/CarouselDistributionComponent.kt

```diff
@@ -192,7 +192,8 @@ private fun getPeekThroughDimension(
                 (viewWidth.toFloat() * (peekThroughSize.value / 100)).dp
         }
         PaddingValues(
-            transformedPeekThroughSize,
+            start = transformedPeekThroughSize,
+            end = transformedPeekThroughSize,
```

Comment:
We can also do it like
```suggestion
            horizontal = transformedPeekThroughSize,
```